eth/stagedsync/stage_mining_exec.go

```diff
@@ -615,107 +595,103 @@ func NotifyPendingLogs(logPrefix string, notifier ChainEventNotifier, logs types
 }
 
 // implemented by tweaking UnwindExecutionStage
-func UnwindMiningExecutionStage(u *UnwindState, s *StageState, tx kv.RwTx, ctx context.Context, cfg MiningExecCfg, logger log.Logger) (err error) {
+func UnwindMiningExecutionStage(u *UnwindState, s *StageState, txc wrap.TxContainer, ctx context.Context, cfg MiningExecCfg, logger log.Logger) (err error) {
```

Comment:
Existing implementation seems to be almost copy of `UnwindExecutionStage` at `stage_execution.go`. So, I merged the new V3's implementation of `UnwindExecutionStage` to here.